app/workers/repository_import_worker.rb

```diff
@@ -7,10 +7,16 @@ class RepositoryImportWorker
   def perform(project_id)
     project = Project.find(project_id)
 
-    import_result = gitlab_shell.send(:import_repository,
+    if project.import_type != 'fogbugz'
```

Comment:
It's better to check the import_url for `http://unknown.git` here, even better if you put that into a constant on Project.